### PR TITLE
Making profile page a header => scrollable

### PR DESCRIPTION
**Pull request by MichaelVargas31**
Got rid of divide between past and future itineraries (still possible to re-add the divide later using headers). Added the profile view to the collection view as a header, made the  collection view span the entire VC. 
Screen Shot 2019-07-23 at 12 22 31 PM

### CODE REVIEW

File: Mytinerary/Controller/ProfileViewController.m [hunk 1]
```diff
@@ -30,6 +31,8 @@ - (void)viewDidLoad {
     self.collectionView.dataSource=self;
     self.collectionView.delegate=self;
     
+    // insert profile cell
```
Comment (fireph) [inline Mytinerary/Controller/ProfileViewController.m:34]: great way to track these kinds of TODOs is to create an issue on Github and link to that issue number in this comment

**Comment by fireph** [comment]
Adding a screenshot or something to give context on what this diff is adding would be helpful for reviewing.

File: Mytinerary/Controller/ProfileViewController.m (hunk 1 above)
Comment (MichaelVargas31) [reply Mytinerary/Controller/ProfileViewController.m:34]: Do you mean link as in referencing the issue number in the comment in the code itself, or is there a way to actually link the two with github?

**Comment by MichaelVargas31** [comment]
Done! Updated the description

**Pull request merged**